syntax = "proto3";

message LocationMessage {


 string person_id = 1;
 string creation_time = 2;
 string latitude = 3;
 string longitude = 4;
}

message Empty {

}


service LocationService {
   rpc Create(LocationMessage) returns (LocationMessage);
}